src/fides/core/config/redis_settings.py

```diff
@@ -13,8 +13,8 @@ class RedisSettings(FidesSettings):
 
     host: str = "redis"
     port: int = 6379
-    user: Optional[str] = ""
-    password: str = "testpassword"
+    user: Optional[str] = None
+    password: str = None
```

Comment:
Defaulting this to `testpassword` is surprising. If I boot up Fides and connect to Redis, I don't expect it to try to authenticate with the password "testpassword" and I'd take forever to figure that out in practice...

Reply:
Should we also mark this as `Optional[str]` since it defaults to `None`?

Reply:
Yes, in fact `mypy` said the same thing 👍

Fixed